website/docs/r/consul_secret_backend_role.html.md

```diff
@@ -43,23 +43,30 @@ The following arguments are supported:
 * `backend` - (Optional) The unique name of an existing Consul secrets backend mount. Must not begin or end with a `/`. One of `path` or `backend` is required.
 
 * `name` - (Required) The name of the Consul secrets engine role to create.
- 
-* `consul_namespace` - (Optional) The Consul namespace that the token will be created in.
-   Applicable for Vault 1.10+ and Consul 1.7+",
-
-* `partition` - (Optional) The admin partition that the token will be created in.
-   Applicable for Vault 1.10+ and Consul 1.11+",
 
-* `policies` - (Required when `consul_roles` is unset) The list of Consul ACL policies to associate with these roles.
+* `consul_policies` - (Required)<sup><a href="#note-about-required-arguments">SEE NOTE</a></sup> The list of Consul ACL policies to associate with these roles.
```

Comment:
I think these fields should still be documented as `Optional`, but we should also make users refer to the note on how at least one of these needs to be provided like you are doing. They are technically all optional but there is a caveat to that